client/my-sites/customer-home/cards/tasks/deprecate-editor/index.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'lib/formatting';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import Task from '../task';
+
+import { setSelectedEditor } from 'state/selected-editor/actions';
+import {
+	composeAnalytics,
+	recordGoogleEvent,
+	recordTracksEvent,
+	withAnalytics,
+	bumpStat,
+} from 'state/analytics/actions';
+import getGutenbergEditorUrl from 'state/selectors/get-gutenberg-editor-url';
+import blockEditorImage from 'assets/images/illustrations/block-editor-fade.svg';
+import FormattedDate from 'components/formatted-date';
+
+const DeprecateEditor = ( { siteId, gutenbergUrl, optIn } ) => {
+	const translate = useTranslate();
+	const actionCallback = () => {
+		optIn( siteId, gutenbergUrl );
+	};
+	return (
+		<Task
+			title={ translate( 'The Block Editor is coming.' ) }
+			description={ preventWidows(
+				translate(
+					'Get a head start before we enable it for everyone on {{strong}}{{date}}{{/date}}{{/strong}}. {{a}}Read more{{/a}}',
+					{
+						components: {
+							a: <a href="#link-yet-to-be-decided" />,
+							strong: <strong />,
+							date: <FormattedDate date="2020-06-01" format="MMMM D" />,
+						},
+					}
+				)
+			) }
+			actionText={ translate( 'Switch to the Block Editor' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **10** times:
`translate( 'Switch to the new block editor.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/home-editor-deprecation-notice)